Back/Reciplastk.Gateway/Services/CustumerService.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Azure;
+using Reciplastk.Gateway.DataAccess;
+using Reciplastk.Gateway.Models;
+using System.Linq;
+
+namespace Reciplastk.Gateway.Services
+{
+    public class CustumerService
+    {
+        private readonly ReciplastkContext db;
+        
+        public CustumerService(ReciplastkContext db)
+        {
+            this.db = db;
+
+        }
+        public HttpResponseModel ShowAllCustomers()
+        {
+            var customer = db.Customers.ToList();
+            var response = new HttpResponseModel();
+            response.WasSuccessful = true;
+            response.Data = customer;
+            return response;
+        }
+        public HttpResponseModel ShowCustomer(int customerid)
+        {
+            var customer = db.Customers.Where(x => x.Customerid == customerid).FirstOrDefault();
+            var response = new HttpResponseModel();
+            if (customer != null)
+            {
+                response.WasSuccessful = true;
+                response.Data = customer;
+            }
+            else
+            {
+                response.WasSuccessful = false;
+                response.StatusMessage = "The customer was not found";
+            }
+            return response;
+        }
+        private Customer CustomerById(int? customerId)
```

Comment:
Remember to name the methods starting with a verb. 
```suggestion
        private Customer GetCustomerById(int? customerId)
```